examples/equilibria/visualisation_of_toroidal_coordinate_transform.ex.py

```diff
@@ -0,0 +1,221 @@
+# ---
```

Comment:
This is really useful but I think should live in documentation rather than an example. In the same form but a page in documentation/source/utilities (which is currently empty).